healthyregions/SDOHPlace-MetadataManager
# #23: Remove the description and schema_provider from suggestion

This PR addresses issue #23 by removing the description and schema_provider fields from the suggested term list.

## How to Test
1. Navigate to https://deploy-preview-359--cheerful-treacle-913a24.netlify.app/search (using the previous PR with the dev core). Type "so" in the search bar. You should now see that long descriptions have been removed from the suggestions:
![Screenshot 2024-11-07 at 5 22 53 PM](https://github.com/user-attachments/assets/7131a456-54a3-4a7b-8aa2-2261914c4319)

2. Type "Health" in the search bar. You should no longer see entries like "Healthy Regions + Policies Lab" containing schema_provider in the list:
![Screenshot 2024-11-07 at 5 23 46 PM](https://github.com/user-attachments/assets/a905f025-7390-40b6-9821-2596f05ff88a)
